scripts/mixins/families/wamouracampa.lua

```diff
@@ -0,0 +1,101 @@
+-- Wamouracampa family mixin
+
+require("scripts/globals/mixins")
+require("scripts/globals/status")
+-----------------------------------
+g_mixins = g_mixins or {}
+g_mixins.families = g_mixins.families or {}
+
+local function curlUpRoaming(mob)
+    mob:setAnimationSub(5) -- Curl
+    mob:setLocalVar("formTimeRoam", os.time() + math.random(43, 47))
+    mob:setMobMod(xi.mobMod.SKILL_LIST, 1162) -- Set Curled Skill List. ("Cannonball" and "Heat Barrier" only)
+    -- TODO: Change resistances.
```

Comment:
I think it would be OK to add placeholder resistances here. I didn't check both wikis, but one did mention that one form is DEF and the other is MDEF.

You could cache the starting DEF/MDEF on spawn, and then use +/- 30% of that for each stat and leave a comment saying that this is a guess.

Reply:
After some testing, we concluded that:
While streched, Phisical Damage Taken = 100%, Magic Damage Taken = 75%
While Curled, Phisical Damage Taken = 75%, Magic Damage Taken = 100%
(And of course, no modifier on the other damage type)